### PR TITLE
groupedlistv2: update getderivedstatefromprops

### LINKED ISSUE #24992: [Bug]: GroupedListV2 Columns Not Aligning
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (6) x64 Intel(R) Core(TM) i5-9600K CPU @ 3.70GHz
    Memory: 5.69 GB / 15.94 GB
  Browsers:
    Chrome: 105.0.5195.127
    Edge: Spartan (44.19041.1266.0), Chromium (105.0.1343.53)
    Internet Explorer: 11.0.19041.1566
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://codepen.io/migee/pen/bGMLqXq

## Actual Behavior
Sizing the columns in the header has no effect on data below. Root level groups and nested groups columns do not align vertically with one another.

## Expected Behavior
Columns across groupings should align vertically

### Logs

_No response_

### Requested priority

Normal

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by spmonahan**
## Current Behavior

`GroupedListV2` does not re-render when props/state that should trigger a re-render are changed (i.e., V2's behavior does not match `GroupedList`).

## New Behavior

`GroupedListV2` re-renders as expected.

## Related Issue(s)

Fixes #24992

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/61a63568f13b2607259997429c51fe8cdc8d95f4'>61a63568f13b2607259997429c51fe8cdc8d95f4</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/BaseButton-mount_base.html">1612</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/BaseButton-mount.html">1606</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Breadcrumb-mount_base.html">3888</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Breadcrumb-mount.html">3708</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Checkbox-mount_base.html">3681</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Checkbox-mount.html">3656</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/CheckboxBase-mount_base.html">3151</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/CheckboxBase-mount.html">3268</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/ChoiceGroup-mount_base.html">6175</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/ChoiceGroup-mount.html">6223</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/ComboBox-mount_base.html">1654</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/ComboBox-mount.html">1655</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/CommandBar-mount_base.html">13135</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/CommandBar-mount.html">13162</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/ContextualMenu-mount_base.html">16175</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/ContextualMenu-mount.html">16481</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/DefaultButton-mount_base.html">1834</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/DefaultButton-mount.html">1974</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/DetailsRow-mount_base.html">5198</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/DetailsRow-mount.html">5234</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/DetailsRowFast-mount_base.html">4863</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/DetailsRowFast-mount.html">5293</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/DetailsRowNoStyles-mount_base.html">4826</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/DetailsRowNoStyles-mount.html">4961</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Dialog-mount_base.html">4253</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Dialog-mount.html">4210</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/DocumentCardTitle-mount_base.html">733</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/DocumentCardTitle-mount.html">740</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Dropdown-mount_base.html">4474</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Dropdown-mount.html">4363</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/FocusTrapZone-mount_base.html">2554</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/FocusTrapZone-mount.html">2642</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/FocusZone-mount_base.html">2572</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/FocusZone-mount.html">2559</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/GroupedList-mount_base.html">75539</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/GroupedList-mount.html">89392</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/GroupedList-virtual-rerender_base.html">35452</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/GroupedList-virtual-rerender.html">34966</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">117351</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/GroupedList-virtual-rerender-with-unmount.html">117579</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/GroupedListV2-mount_base.html">702</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/GroupedListV2-mount.html">730</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/GroupedListV2-virtual-rerender_base.html">673</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/GroupedListV2-virtual-rerender.html">747</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">716</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">750</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/IconButton-mount_base.html">2628</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/IconButton-mount.html">2615</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Label-mount_base.html">930</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Label-mount.html">1011</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Layer-mount_base.html">5878</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Layer-mount.html">5816</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Link-mount_base.html">1104</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Link-mount.html">1111</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/MenuButton-mount_base.html">2474</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/MenuButton-mount.html">2388</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/MessageBar-mount_base.html">3101</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/MessageBar-mount.html">3176</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Nav-mount_base.html">4694</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Nav-mount.html">4427</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/OverflowSet-mount_base.html">1827</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/OverflowSet-mount.html">1794</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Panel-mount_base.html">3372</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Panel-mount.html">3292</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Persona-mount_base.html">1723</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Persona-mount.html">1788</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Pivot-mount_base.html">2310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Pivot-mount.html">2241</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/PrimaryButton-mount_base.html">2200</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/PrimaryButton-mount.html">2096</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Rating-mount_base.html">10228</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Rating-mount.html">10110</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/SearchBox-mount_base.html">2025</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/SearchBox-mount.html">2186</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Shimmer-mount_base.html">3900</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Shimmer-mount.html">4093</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Slider-mount_base.html">2914</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Slider-mount.html">2868</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/SpinButton-mount_base.html">6373</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/SpinButton-mount.html">6327</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Spinner-mount_base.html">1056</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Spinner-mount.html">969</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/SplitButton-mount_base.html">4054</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/SplitButton-mount.html">4316</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Stack-mount_base.html">1212</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Stack-mount.html">1171</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/StackWithIntrinsicChildren-mount_base.html">3423</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/StackWithIntrinsicChildren-mount.html">3266</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/StackWithTextChildren-mount_base.html">6973</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/StackWithTextChildren-mount.html">7254</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/SwatchColorPicker-mount_base.html">14274</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/SwatchColorPicker-mount.html">14338</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/TagPicker-mount_base.html">3719</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/TagPicker-mount.html">3574</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/TeachingBubble-mount_base.html">117545</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/TeachingBubble-mount.html">119092</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Text-mount_base.html">1053</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Text-mount.html">1018</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/TextField-mount_base.html">2206</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/TextField-mount.html">2146</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/ThemeProvider-mount_base.html">1980</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/ThemeProvider-mount.html">1967</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/ThemeProvider-virtual-rerender_base.html">1380</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/ThemeProvider-virtual-rerender.html">1429</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2750</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2863</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Toggle-mount_base.html">1505</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/Toggle-mount.html">1493</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/buttonNative-mount_base.html">703</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25301/perf-test/buttonNative-mount.html">704</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by spmonahan** [comment]
I accepted all the Screener changes as they were just due to images loading/not loading in the test run.

**Review by behowell — APPROVED** [review]

File: packages/react/src/components/GroupedList/GroupedListV2.base.tsx [hunk 1]
```diff
@@ -517,16 +518,25 @@ export class GroupedListV2Wrapper
...
-    const { groups } = nextProps;
+    const { groups, selectionMode, compact, items, listProps } = nextProps;
+    const nextListVersion = listProps && listProps.version;
 
-    if (groups !== previousState.groups) {
-      return {
-        ...previousState,
-        groups,
-      };
+    let nextState = {
```
Comment (spmonahan) [inline packages/react/src/components/GroupedList/GroupedListV2.base.tsx:524]:
```suggestion
    const nextState = {
```

**Review by sopranopillow — APPROVED** [review]

**Pull request merged**